packages/cli/src/parallel-tests/__tests__/merge-test-results.spec.ts

```diff
@@ -121,4 +123,16 @@ describe("mergeResults", () => {
       ])
     );
   });
+
+  it("returns a fail outcome if the original outcome was a pass and there are flakes", () => {
```

Comment:
Isn't this basically identical to the "doesn't ignore diffs to screenshots which originally passed" test case, but with the `noDiff(0), diff(1)` order swapped? Wonder if we could just delete the "doesn't ignore diffs to screenshots which originally passed" test case.

Reply:
oops that's true. will drop this test